.github/workflows/run_tests.yml

```diff
@@ -108,15 +108,15 @@ jobs:
       - name: Install SmartSim (with ML backends)
         run: |
           python -m pip install git+https://github.com/CrayLabs/SmartRedis.git@develop#egg=smartredis
-          python -m pip install .[dev,ml]
+          python -m pip install .[dev]
```

Comment:
Could we throw a deprecation warning and still do a no-op if someone requests 'ml', since this is a change in behavior for users